GovUK-Onboarding.xcworkspace/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>FILEHEADER</key>
+	<string>
+// GOV.UK
+//
+// Copyright © ___YEAR___ ___ORGANIZATIONNAME___.
+// All Rights Reserved.
+//</string>
```

Comment:
Might be something to take advice on - if you haven't already. We usually let the MIT license take care of this I think.